### ISSUE TITLE
Don't warn on EmptySecondaryConstructor if it is calling another constructor.

### ISSUE THREAD

**Issue by yogurtearl** [detekt/detekt#3126]
## Expected Behavior of the rule

`EmptySecondaryConstructor` should not trigger on this constructor. 

This construction has meaningful code, by calling the primary constructor `this(parcel.readString())` 

```kotlin
private data class Foo(val bar: String?) : Parcelable {
    constructor(parcel: Parcel) : this(parcel.readString()) {}
}
```

## Context

The rule should trigger unless really does nothing at all.

**Comment by osipxd** [comment]
Try to remove unnecessary `{}` after `this(...)`

**Comment by BraisGabin** [comment]
The @osipxd's comment should fix this issue. Don't hestiate to comment/open if it doesn't work.
